fullnode/src/main.rs

```diff
@@ -1,19 +1,18 @@
-#[macro_use]
 extern crate serde_json;
 
 use clap::{crate_version, App, Arg};
 use log::*;
 
 use solana::client::mk_client;
 use solana::cluster_info::{Node, NodeInfo, FULLNODE_PORT_RANGE};
-use solana::create_vote_account;
 use solana::fullnode::{Fullnode, FullnodeReturnType};
 use solana::leader_scheduler::LeaderScheduler;
-use solana::rpc_request::{RpcClient, RpcRequest};
+use solana::local_vote_signer_service::*;
```

Comment:
Following the "When in Rome..." principle, can you be explicit with your imports? You use that wildcard another half dozen times below too.